output.go

```diff
@@ -276,5 +276,22 @@ func (o *OutputAccount) MergeStorageUpdates(outAcc *OutputAccount) {
 	}
 }
 
+// GetFirstDataItem returns the first item from the Data field of a LogEntry
```

Comment:
you should move this function where the structure is defined. Also why this function is needed ? 
We discussed letting the data field as a `[]byte` not `[][]byte`